docs/trace/building-your-own-exporter/Program.cs

```diff
@@ -24,12 +24,13 @@ public class Program
 
     public static void Main()
     {
-        using var tracerProvider = Sdk.CreateTracerProvider(
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddActivitySources(
             new string[]
             {
                 "MyCompany.MyProduct.MyLibrary",
             })
-            .AddMyExporter();
+            .Build().AddMyExporter();

```

Comment:
I know you can add exporter after build, but hey 😺

Reply:
Yes, for builder, it makes more sense to add everything before `Build()`.